app/scripts/background.js

```diff
@@ -751,12 +754,31 @@ async function openPopup() {
   });
 }
 
+// It adds the "App Installed" event into a queue of events, which will be tracked only after a user opts into metrics.
+const addAppInstalledEvent = () => {
+  if (controller) {
+    controller.metaMetricsController.addEventBeforeMetricsOptIn({
+      category: EVENT.CATEGORIES.APP,
+      event: EVENT_NAMES.APP_INSTALLED,
+      properties: {
+        signup_date: new Date().toISOString().split('T')[0], // yyyy-mm-dd
```

Comment:
@dan437 and i already discussed we wanted to move this property, `signup_date`, out of this event and into a user trait. properties list should be `{}` here